metatron/src/main/java/uk/gov/ida/eidas/metatron/core/dto/EidasCountryConfig.java

```diff
@@ -0,0 +1,87 @@
+package uk.gov.ida.eidas.metatron.core.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
+public class EidasCountryConfig {
+
+    @JsonProperty
+    @Valid
+    @NotNull
+    private String name;
+    @JsonProperty
+    @Valid
+    @NotNull
+    private String countryCode;
+    @JsonProperty
+    @Valid
+    @NotNull
+    private String connectorMetadata;
+    @JsonProperty
+    @Valid
+    @NotNull
+    private boolean enabled;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getCountryCode() {
+        return countryCode;
+    }
+
+    public void setCountryCode(String countryCode) {
+        this.countryCode = countryCode;
+    }
+
+    @JsonIgnore
+    public String getEntityId() {
+        return connectorMetadata;
+    }
+
+    public String getConnectorMetadata() {
+        return connectorMetadata;
+    }
+
+    public void setConnectorMetadata(String connectorMetadata) {
+        this.connectorMetadata = connectorMetadata;
+    }
+
+    public boolean isEnabled() {
+        return enabled;
+    }
+
+    public void setEnabled(boolean enabled) {
+        this.enabled = enabled;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
My personal inclination would be to write this as `toHtml()` and then if necessary to override `toString`, have it `return toHtml();` inside of it. Purely for clarity - I don't think it's very important.

Reply:
This isn't a thing that will exist in the new world, it was just to get some content out of the app

Reply:
This was just a throw away thing that wasn't intended to live past the basic skeleton app which was only ever really to show the deploy stuff working and the config being picked up.  It will undoubtedly change in the future.